src/Scratch.as

```diff
@@ -466,6 +468,7 @@ public class Scratch extends Sprite {
 		removeLoadProgressBox();
 		System.gc();
 		if (autostart) runtime.startGreenFlags(true);
+		loadInProgress = false;
```

Comment:
ScratchRuntime::installProject() is the only caller to projectLoaded() and installNewProject() calls installProject. Scratch's projectLoaded() and loadProjectFailed() are now the only places that loadInProgress are set to false. It's cleaner and clearer this way.